cli.js

```diff
@@ -121,6 +121,12 @@ program
 
 							writeStream.write((wroteData ? '\n' : '') + data.join('\n'));
 							break;
+
+						case 'stdout':
+							data = _.map(data, function(row) {
+								console.log(row.ipAddress + ':' + row.port);
+							});
```

Comment:
I don't think `console.log` is the right way to output to stdout. Should probably use `process.stdout.write(row.ipAddress + ':' + row.port + '\n')`.